pkg/loop/internal/chain_reader_test.go

```diff
@@ -2,17 +2,19 @@ package internal
 
 import (
 	"errors"
-	"github.com/stretchr/testify/assert"
```

Comment:
Lint fixed this in this pr but not the other, not sure why, but I don't feel like it needs to be reverted...